### DISCUSSION TITLE
Option to change pan/move gesture behavior to have inertia/to decelerate

### DISCUSSION THREAD

**Discussion by LucasBerger** [rnmapbox/maps#2763] [Ideas]
**Describe the bug**  
is there an option to use panDeceleration as described in the mapbox docs ([configure-pan-behavior](https://docs.mapbox.com/ios/maps/guides/user-interaction/#configure-pan-behavior))?
Or something that produces the same effect?

**To Reproduce**  
none

**Expected behavior**  
be able to change the pan behavior to have inertia or deceleration after dragging stops

**Actual behavior**  
no option at this time exposed

**Versions (please complete the following information):**  
 - Platform: [Android, iOS]
 - @rnmapbox/maps Version [v10 latest]
 - React Native Version [0.69.1]

**Comment by mfazekas** [comment]
I see panDecelerationFactor on iOS but not on android.

https://docs.mapbox.com/ios/maps/api/10.7.0/Structs/GestureOptions.html#/s:10MapboxMaps14GestureOptionsV21panDecelerationFactor12CoreGraphics7CGFloatVvp

PR wellcome!
